charts/incubator/hyperswitch-helm/values.yaml

```
@@ -301,27 +301,37 @@ redis:
 # TODO to handle
 externalRedis: {}
 
+postgresql:
+  enabled: true
+  global:
+    postgresql:
+      auth:
+        username: "hyperswitch"
```

by default db_user is used in router
```suggestion
        username: "db_user"
```